test/unit/quantum/results/test_reckoning.py

```diff
@@ -32,40 +32,76 @@ class TestExpvalReckoner:
     """Test ExpvalReckoner interface."""
 
     @mark.parametrize(
-        "counts, expected",
+        "data, expected",
```

Comment:
I see where you are coming from, don't you think that this would be better though?
```suggestion
        "frequencies, expected",
```